packages/test/generatedCode/apiClient.ts

```diff
@@ -1,57 +1,43 @@
+                import {FormParseNodeFactory, FormSerializationWriterFactory} from '@microsoft/kiota-serialization-form';
+                import {JsonParseNodeFactory, JsonSerializationWriterFactory} from '@microsoft/kiota-serialization-json';
+                import {TextParseNodeFactory, TextSerializationWriterFactory} from '@microsoft/kiota-serialization-text';
 
-/**
- * The main entry point of the SDK, exposes the configuration and the fluent API.
- */
-export class ApiClient {
-    /** Path parameters for the request */
-    /** Path parameters for the request */
-    private pathParameters: Record<string, unknown>;
-    /** The request adapter to use to execute the requests. */
-    /** The request adapter to use to execute the requests. */
-    private requestAdapter: RequestAdapter;
-    /** Url template to use to build the URL for the current request builder */
-    /** Url template to use to build the URL for the current request builder */
-    private urlTemplate: string;
-    /** The users property */
-    /** The users property */
-    public get users(): UsersRequestBuilder {
-        return new UsersRequestBuilder(this.pathParameters, this.requestAdapter);
-    }
-    /**
-     * Instantiates a new ApiClient and sets the default values.
-     * @param requestAdapter The request adapter to use to execute the requests.
-     */
-    public constructor(requestAdapter: RequestAdapter) {
-        if(!requestAdapter) throw new Error("requestAdapter cannot be undefined");
-        this.pathParameters = {};
-        this.urlTemplate = "{+baseurl}";
-        this.requestAdapter = requestAdapter;
-        registerDefaultSerializer(JsonSerializationWriterFactory);
-        registerDefaultSerializer(TextSerializationWriterFactory);
-        registerDefaultSerializer(FormSerializationWriterFactory);
-        registerDefaultDeserializer(JsonParseNodeFactory);
-        registerDefaultDeserializer(TextParseNodeFactory);
-        registerDefaultDeserializer(FormParseNodeFactory);
-        if (requestAdapter.baseUrl === undefined || requestAdapter.baseUrl === "") {
-            requestAdapter.baseUrl = "https://graph.microsoft.com/v1.0";
-        }
-        this.pathParameters["baseurl"] = requestAdapter.baseUrl;
-    };
-    /**
-     * Gets an item from the ApiSdk.users.item collection
-     * @param id Unique identifier of the item
-     * @returns a UserItemRequestBuilder
-     */
-    public usersById(id: string) : UserItemRequestBuilder {
-        if(!id) throw new Error("id cannot be undefined");
-        const urlTplParams = getPathParameters(this.pathParameters);
-        urlTplParams["user%2Did"] = id
-        return new UserItemRequestBuilder(urlTplParams, this.requestAdapter);
-    };
-}
+                /**
+                 * The main entry point of the SDK, exposes the configuration and the fluent API.
+                 */
+                export class ApiClient {
+                    /** Path parameters for the request */
```

Comment:
noting that here knowing it may come from the parent PR, but the doc comments are being inserted twice.